tests/testsuite/cargo_remove/invalid_gc_patch/out/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+# Cargo.toml
+
+[workspace]
+members = ["serde", "serde_derive"]
+
+[patch.crates-io]
+serde = { path = "serde" }
+
```

Comment:
> [chore: add failing test](https://github.com/rust-lang/cargo/pull/12454/commits/9d3fcfab35ce0c063c49a8a5ab19fb16646a9f7b)

Please make the commit that adds the test pass.  Then when you have a commit that changes the behavior, this test will then need to be updated.  As a reviewer, I can then see how the behavior changes with the code change.  This also makes it nicer for bisections as the tests are always passing.